### PR TITLE
DOP-2319 - Add new quiz directives to snooty-parser

### PR DESCRIPTION
**Pull request by carooobregon**
Stories/Links:
DOP-2319

Notes:
Added new directives to snooty-parser to support the new Quiz Widget

### CODE REVIEW

**Comment by i80and** [comment]
@carooobregon could you run `make format` and commit? CI is failing

File: snooty/rstspec.toml [hunk 1]
```diff
@@ -707,6 +707,26 @@ example = """.. procedure::
...
+   
+   ${2:string}"""
+argument_type = "string"
+options.quizid = {type = "string", required = true}
+
+[directive."mongodb:quizchoice"]
+content_type = "block"
+help = "A possible quiz answer and an explanation to be displayed after submission"
+example = """ .. quizchoice:: ${1:string}
+   :istrue:
```
Comment (mango-db) [inline snooty/rstspec.toml:724]: Nit: do we want to perhaps use kebab-case `is-true` like [so](https://github.com/mongodb/snooty-parser/blob/master/snooty/rstspec.toml#L696), or just `true` like [so](https://github.com/mongodb/snooty-parser/blob/master/snooty/rstspec.toml#L314)? I have no personal preferences

Comment (carooobregon) [reply snooty/rstspec.toml:724]: I personally like `is-true` but open to both! Changing it unless @i80and has any personal preference

Comment (i80and) [reply snooty/rstspec.toml:724]: I like kebab case

Comment (i80and) [reply snooty/rstspec.toml:724]: quiz-id should probably also get the kebab treatment if we do this

**Review by i80and — APPROVED** [review]

**Pull request merged**